sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -286,13 +292,13 @@ def _create_handler(self, auth):
             auto_complete=False,
             encoding=self._config.encoding,
             receive_settle_mode=ServiceBusToAMQPReceiveModeMap[self._receive_mode],
-            send_settle_mode=SenderSettleMode.Settled \
-                if self._receive_mode == ServiceBusReceiveMode.RECEIVE_AND_DELETE \
-                else None,
+            send_settle_mode=SenderSettleMode.Settled
+            if self._receive_mode == ServiceBusReceiveMode.RECEIVE_AND_DELETE
+            else None,
```

Comment:
Interesting... I didn't know it was valid to have the if/else on the same indentation like this :)

Reply:
welcome to the 2077 python world :P